chroot-etc/croutonwheel.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
```

Comment:
Can't this be automatically started in croutonxinitrc-wrapper?  The same place all the synclient stuff got if'd out.